### PR TITLE
Adding Codebyte for getUTCHours

### PR DESCRIPTION
**Pull request by bpicoCode**
### Description
- Adding Codebyte for .getUTCHours() entry
- Closes issue #4150  

### Type of Change
- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/javascript/concepts/dates/terms/getUTCHours/getUTCHours.md [hunk 1]
```diff
@@ -35,3 +35,15 @@ Returns the following to the console:
 ```shell
 2
 ```
+
+### Codebyte Example
+In the following example, variables `estDate` and `pstDate` are defined to store a new `Date` object in Eastern Time and Pacific Time respectively. The method `.getUTCHours()` is then used to print the hour of the dates according to universal time. Run this example in the Codebyte to test the results.
```
Comment (avdhoottt) [inline content/javascript/concepts/dates/terms/getUTCHours/getUTCHours.md:40]: Give space before and after the code byte.

File: content/javascript/concepts/dates/terms/getUTCHours/getUTCHours.md [hunk 2]
```diff
@@ -35,3 +35,15 @@ Returns the following to the console:
 ```shell
 2
 ```
+
+### Codebyte Example
```
Comment (avdhoottt) [inline content/javascript/concepts/dates/terms/getUTCHours/getUTCHours.md:39]:
```suggestion
## Codebyte Example
```
It should be a H2 subheading.

File: content/javascript/concepts/dates/terms/getUTCHours/getUTCHours.md (hunk 1 above)
Comment (avdhoottt) [inline content/javascript/concepts/dates/terms/getUTCHours/getUTCHours.md:40]:
```suggestion
In the following example, variables `estDate` and `pstDate` are defined to store a new `Date` object in Eastern Time and Pacific Time respectively. The method `.getUTCHours()` is then used to print the hours of the dates according to universal time. You can just run this example in the Codebyte to test the results. 
```

**Review by avdhoottt — CHANGES_REQUESTED** [review]
@bpicoCode Please make the mentioned changes.

**Comment by bpicoCode** [comment]
I made the changes as requested. Let me know if there is anything else I need to do. Thank you!

File: content/javascript/concepts/dates/terms/getUTCHours/getUTCHours.md [hunk 3]
```diff
@@ -35,3 +35,16 @@ Returns the following to the console:
 ```shell
 2
 ```
+
+## Codebyte Example
+
+In the following example, variables `estDate` and `pstDate` are defined to store a new `Date` object in Eastern Time and Pacific Time respectively. The method `.getUTCHours()` is then used to print the hours of the dates according to universal time. You can just run this example in the Codebyte to test the results.
```
Comment (avdhoottt) [inline content/javascript/concepts/dates/terms/getUTCHours/getUTCHours.md:41]:
```suggestion
In the following example, variables `estDate` and `pstDate` are defined to store a new `Date` object in Eastern Time and Pacific Time respectively. The method `.getUTCHours()` is then used to print the hours of the dates according to universal time. You can just run this example in the Codebyte to test the results. 

```

**Review by avdhoottt — APPROVED** [review]
It's ready for the second review. @bpicoCode Please sign our CLA as it is mandatory for authors.

**Review by dakshdeepHERE — APPROVED** [review]
LGTM for merge 🚀

**Comment by dakshdeepHERE** [comment]
@bpicoCode can you please sign the CLA assistant so we can merge this PR?

**Comment by bpicoCode** [comment]
> @bpicoCode can you please sign the CLA assistant so we can merge this PR?
I signed the CLA but my initial commit came from my local repo which wasn't linked to a Github user. See that 'Benjamin Picone' hasn't signed yet but bpicoCode (also me) has signed. I can't change that initial commit. Maybe we should delete that commit so that 'Benjamin Picone' no longer shows up?

**Comment by bpicoCode** [comment]
@dakshdeepHERE Alternatively we could create a new pull request and link this one. I configured my Githib

> > @bpicoCode can you please sign the CLA assistant so we can merge this PR?
> > I signed the CLA but my initial commit came from my local repo which wasn't linked to a Github user. See that 'Benjamin Picone' hasn't signed yet but bpicoCode (also me) has signed. I can't change that initial commit. Maybe we should delete that commit so that 'Benjamin Picone' no longer shows up?

Alternatively we could create a new pull request with all the review changes.

**Comment by dakshdeepHERE** [comment]
@bpicoCode please go on

**Comment by dakshdeepHERE** [comment]
This PR is getting closed as the new one is opened by the Author.

**Pull request closed without merging**